### ISSUE TITLE
Crash when going in nether

### ISSUE THREAD

**Issue by UnPainPerdu** [Asek3/Oculus#387]
### Minecraft Version

Minecraft 1.20.1

### Oculus Version

oculus-mc1.20-1.6.4

### Sodium Version

rubidium-0.6.5

### Operating System

Windows 10

### What is your GPU?

NVIDIA GeForce RTX 2070 SUPER

### Java Version

default for 1.20.1

### Reproduction Steps

1. Build a nether portal
2. Go in it
3. Game Crashes when arrive in nether

### Crash Report file and latest.log

```shell
---- Minecraft Crash Report ----
// This doesn't make any sense!

Time: 2023-06-23 11:57:52
Description: Unexpected error

java.lang.NullPointerException: Cannot invoke "net.minecraft.client.renderer.RenderType.m_110185_()" because "type" is null
	at net.coderbot.batchedentityrendering.impl.FullyBufferedMultiBufferSource.m_109911_(FullyBufferedMultiBufferSource.java:136) ~[oculus-mc1.20-1.6.4.jar%23162!/:?] {re:mixin,re:classloading}
	at net.minecraft.client.renderer.LevelRenderer.m_109599_(LevelRenderer.java:1377) ~[client-1.20.1-20230612.114412-srg.jar%23164!/:?] {re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:classloading,pl:accesstransformer:B,pl:mixin:APP:mixins.oculus.vertexformat.json:immediate.MixinLevelRenderer,pl:mixin:APP:oculus-batched-entity-rendering.mixins.json:MixinLevelRenderer,pl:mixin:APP:oculus-batched-entity-rendering.mixins.json:MixinLevelRenderer_EntityListSorting,pl:mixin:APP:rubidium.mixins.json:features.buffer_builder.intrinsics.MixinWorldRenderer,pl:mixin:APP:rubidium.mixins.json:features.chunk_rendering.MixinWorldRenderer,pl:mixin:APP:rubidium.mixins.json:features.clouds.MixinWorldRenderer,pl:mixin:APP:rubidium.mixins.json:features.options.MixinWorldRenderer,pl:mixin:APP:mixins.oculus.json:LevelRendererAccessor,pl:mixin:APP:mixins.oculus.json:MixinLevelRenderer,pl:mixin:APP:mixins.oculus.json:fabulous.MixinDisableFabulousGraphics,pl:mixin:APP:mixins.oculus.json:shadows.MixinLevelRenderer,pl:mixin:APP:mixins.oculus.json:sky.MixinLevelRenderer_SunMoonToggle,pl:mixin:APP:mixins.oculus.fantastic.json:MixinLevelRenderer,pl:mixin:APP:mixins.oculus.compat.sodium.json:sky.MixinLevelRenderer,pl:mixin:APP:mixins.oculus.json:shadows.MixinPreventRebuildNearInShadowPass,pl:mixin:A,pl:runtimedistcleaner:A}
	at net.minecraft.client.renderer.GameRenderer.m_109089_(GameRenderer.java:1126) ~[client-1.20.1-20230612.114412-srg.jar%23164!/:?] {re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:classloading,pl:accesstransformer:B,pl:mixin:APP:mixins.oculus.json:GameRendererAccessor,pl:mixin:APP:mixins.oculus.json:MixinGameRenderer,pl:mixin:APP:mixins.oculus.json:MixinModelViewBobbing,pl:mixin:APP:mixins.oculus.json:MixinTweakFarPlane,pl:mixin:APP:mixins.oculus.json:MixinGameRenderer_NightVisionCompat,pl:mixin:A,pl:runtimedistcleaner:A}
	at net.minecraft.client.renderer.GameRenderer.m_109093_(GameRenderer.java:909) ~[client-1.20.1-20230612.114412-srg.jar%23164!/:?] {re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:classloading,pl:accesstransformer:B,pl:mixin:APP:mixins.oculus.json:GameRendererAccessor,pl:mixin:APP:mixins.oculus.json:MixinGameRenderer,pl:mixin:APP:mixins.oculus.json:MixinModelViewBobbing,pl:mixin:APP:mixins.oculus.json:MixinTweakFarPlane,pl:mixin:APP:mixins.oculus.json:MixinGameRenderer_NightVisionCompat,pl:mixin:A,pl:runtimedistcleaner:A}
	at net.minecraft.client.Minecraft.m_91383_(Minecraft.java:1146) ~[client-1.20.1-20230612.114412-srg.jar%23164!/:?] {re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:classloading,pl:accesstransformer:B,pl:mixin:APP:rubidium.mixins.json:core.MixinMinecraftClient,pl:mixin:APP:mixins.oculus.json:MixinMinecraft_Keybinds,pl:mixin:APP:mixins.oculus.json:MixinMinecraft_Images,pl:mixin:APP:mixins.oculus.json:MixinMinecraft_PipelineManagement,pl:mixin:A,pl:runtimedistcleaner:A}
	at net.minecraft.client.Minecraft.m_91374_(Minecraft.java:718) ~[client-1.20.1-20230612.114412-srg.jar%23164!/:?] {re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:classloading,pl:accesstransformer:B,pl:mixin:APP:rubidium.mixins.json:core.MixinMinecraftClient,pl:mixin:APP:mixins.oculus.json:MixinMinecraft_Keybinds,pl:mixin:APP:mixins.oculus.json:MixinMinecraft_Images,pl:mixin:APP:mixins.oculus.json:MixinMinecraft_PipelineManagement,pl:mixin:A,pl:runtimedistcleaner:A}
	at net.minecraft.client.main.Main.main(Main.java:218) ~[forge-47.0.6.jar:?] {re:classloading,pl:runtimedistcleaner:A}
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?] {}
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?] {}
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?] {}
	at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?] {}
	at net.minecraftforge.fml.loading.targets.CommonLaunchHandler.runTarget(CommonLaunchHandler.java:111) ~[fmlloader-1.20.1-47.0.6.jar:?] {}
	at net.minecraftforge.fml.loading.targets.CommonLaunchHandler.clientService(CommonLaunchHandler.java:99) ~[fmlloader-1.20.1-47.0.6.jar:?] {}
	at net.minecraftforge.fml.loading.targets.CommonClientLaunchHandler.lambda$makeService$0(CommonClientLaunchHandler.java:25) ~[fmlloader-1.20.1-47.0.6.jar:?] {}
	at cpw.mods.modlauncher.LaunchServiceHandlerDecorator.launch(LaunchServiceHandlerDecorator.java:30) ~[modlauncher-10.0.9.jar:?] {}
	at cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:53) ~[modlauncher-10.0.9.jar:?] {}
	at cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:71) ~[modlauncher-10.0.9.jar:?] {}
	at cpw.mods.modlauncher.Launcher.run(Launcher.java:108) ~[modlauncher-10.0.9.jar:?] {}
	at cpw.mods.modlauncher.Launcher.main(Launcher.java:78) ~[modlauncher-10.0.9.jar:?] {}
	at cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:26) ~[modlauncher-10.0.9.jar:?] {}
	at cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:23) ~[modlauncher-10.0.9.jar:?] {}
	at cpw.mods.bootstraplauncher.BootstrapLauncher.main(BootstrapLauncher.java:141) ~[bootstraplauncher-1.1.2.jar:?] {}


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Render thread
Stacktrace:
	at net.coderbot.batchedentityrendering.impl.FullyBufferedMultiBufferSource.m_109911_(FullyBufferedMultiBufferSource.java:136) ~[oculus-mc1.20-1.6.4.jar%23162!/:?] {re:mixin,re:classloading}
	at net.minecraft.client.renderer.LevelRenderer.m_109599_(LevelRenderer.java:1377) ~[client-1.20.1-20230612.114412-srg.jar%23164!/:?] {re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:classloading,pl:accesstransformer:B,pl:mixin:APP:mixins.oculus.vertexformat.json:immediate.MixinLevelRenderer,pl:mixin:APP:oculus-batched-entity-rendering.mixins.json:MixinLevelRenderer,pl:mixin:APP:oculus-batched-entity-rendering.mixins.json:MixinLevelRenderer_EntityListSorting,pl:mixin:APP:rubidium.mixins.json:features.buffer_builder.intrinsics.MixinWorldRenderer,pl:mixin:APP:rubidium.mixins.json:features.chunk_rendering.MixinWorldRenderer,pl:mixin:APP:rubidium.mixins.json:features.clouds.MixinWorldRenderer,pl:mixin:APP:rubidium.mixins.json:features.options.MixinWorldRenderer,pl:mixin:APP:mixins.oculus.json:LevelRendererAccessor,pl:mixin:APP:mixins.oculus.json:MixinLevelRenderer,pl:mixin:APP:mixins.oculus.json:fabulous.MixinDisableFabulousGraphics,pl:mixin:APP:mixins.oculus.json:shadows.MixinLevelRenderer,pl:mixin:APP:mixins.oculus.json:sky.MixinLevelRenderer_SunMoonToggle,pl:mixin:APP:mixins.oculus.fantastic.json:MixinLevelRenderer,pl:mixin:APP:mixins.oculus.compat.sodium.json:sky.MixinLevelRenderer,pl:mixin:APP:mixins.oculus.json:shadows.MixinPreventRebuildNearInShadowPass,pl:mixin:A,pl:runtimedistcleaner:A}
	at net.minecraft.client.renderer.GameRenderer.m_109089_(GameRenderer.java:1126) ~[client-1.20.1-20230612.114412-srg.jar%23164!/:?] {re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:classloading,pl:accesstransformer:B,pl:mixin:APP:mixins.oculus.json:GameRendererAccessor,pl:mixin:APP:mixins.oculus.json:MixinGameRenderer,pl:mixin:APP:mixins.oculus.json:MixinModelViewBobbing,pl:mixin:APP:mixins.oculus.json:MixinTweakFarPlane,pl:mixin:APP:mixins.oculus.json:MixinGameRenderer_NightVisionCompat,pl:mixin:A,pl:runtimedistcleaner:A}
-- Affected level --
Details:
	All players: 1 total; [LocalPlayer['Un_Pain_Perdu'/56, l='ClientLevel', x=-1013.05, y=117.00, z=185.99]]
	Chunk stats: 4096, 22
	Level dimension: minecraft:the_nether
	Level spawn location: World: (0,65,0), Section: (at 0,1,0 in 0,4,0; chunk contains blocks 0,0,0 to 15,255,15), Region: (0,0; contains chunks 0,0 to 31,31, blocks 0,0,0 to 511,255,511)
	Level time: 2591299 game time, 3218050 day time
	Server brand: forge
	Server type: Integrated singleplayer server
Stacktrace:
	at net.minecraft.client.multiplayer.ClientLevel.m_6026_(ClientLevel.java:455) ~[client-1.20.1-20230612.114412-srg.jar%23164!/:?] {re:mixin,pl:runtimedistcleaner:A,re:classloading,pl:mixin:APP:rubidium.mixins.json:features.chunk_rendering.MixinClientWorld,pl:mixin:APP:rubidium.mixins.json:features.fast_biome_colors.MixinClientWorld,pl:mixin:APP:mixins.oculus.vertexformat.json:block_rendering.MixinClientLevel,pl:mixin:A,pl:runtimedistcleaner:A}
	at net.minecraft.client.Minecraft.m_91354_(Minecraft.java:2319) ~[client-1.20.1-20230612.114412-srg.jar%23164!/:?] {re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:classloading,pl:accesstransformer:B,pl:mixin:APP:rubidium.mixins.json:core.MixinMinecraftClient,pl:mixin:APP:mixins.oculus.json:MixinMinecraft_Keybinds,pl:mixin:APP:mixins.oculus.json:MixinMinecraft_Images,pl:mixin:APP:mixins.oculus.json:MixinMinecraft_PipelineManagement,pl:mixin:A,pl:runtimedistcleaner:A}
	at net.minecraft.client.Minecraft.m_91374_(Minecraft.java:740) ~[client-1.20.1-20230612.114412-srg.jar%23164!/:?] {re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:classloading,pl:accesstransformer:B,pl:mixin:APP:rubidium.mixins.json:core.MixinMinecraftClient,pl:mixin:APP:mixins.oculus.json:MixinMinecraft_Keybinds,pl:mixin:APP:mixins.oculus.json:MixinMinecraft_Images,pl:mixin:APP:mixins.oculus.json:MixinMinecraft_PipelineManagement,pl:mixin:A,pl:runtimedistcleaner:A}
	at net.minecraft.client.main.Main.main(Main.java:218) ~[forge-47.0.6.jar:?] {re:classloading,pl:runtimedistcleaner:A}
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?] {}
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?] {}
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?] {}
	at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?] {}
	at net.minecraftforge.fml.loading.targets.CommonLaunchHandler.runTarget(CommonLaunchHandler.java:111) ~[fmlloader-1.20.1-47.0.6.jar:?] {}
	at net.minecraftforge.fml.loading.targets.CommonLaunchHandler.clientService(CommonLaunchHandler.java:99) ~[fmlloader-1.20.1-47.0.6.jar:?] {}
	at net.minecraftforge.fml.loading.targets.CommonClientLaunchHandler.lambda$makeService$0(CommonClientLaunchHandler.java:25) ~[fmlloader-1.20.1-47.0.6.jar:?] {}
	at cpw.mods.modlauncher.LaunchServiceHandlerDecorator.launch(LaunchServiceHandlerDecorator.java:30) ~[modlauncher-10.0.9.jar:?] {}
	at cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:53) ~[modlauncher-10.0.9.jar:?] {}
	at cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:71) ~[modlauncher-10.0.9.jar:?] {}
	at cpw.mods.modlauncher.Launcher.run(Launcher.java:108) ~[modlauncher-10.0.9.jar:?] {}
	at cpw.mods.modlauncher.Launcher.main(Launcher.java:78) ~[modlauncher-10.0.9.jar:?] {}
	at cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:26) ~[modlauncher-10.0.9.jar:?] {}
	at cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:23) ~[modlauncher-10.0.9.jar:?] {}
	at cpw.mods.bootstraplauncher.BootstrapLauncher.main(BootstrapLauncher.java:141) ~[bootstraplauncher-1.1.2.jar:?] {}


-- Last reload --
Details:
	Reload number: 1
	Reload reason: initial
	Finished: Yes
	Packs: mod_resources, vanilla

-- System Details --
Details:
	Minecraft Version: 1.20.1
	Minecraft Version ID: 1.20.1
	Operating System: Windows 10 (amd64) version 10.0
	Java Version: 17.0.3, Microsoft
	Java VM Version: OpenJDK 64-Bit Server VM (mixed mode), Microsoft
	Memory: 1951556768 bytes (1861 MiB) / 3388997632 bytes (3232 MiB) up to 17179869184 bytes (16384 MiB)
	CPUs: 12
	Processor Vendor: AuthenticAMD
	Processor Name: AMD Ryzen 5 3600XT 6-Core Processor            
	Identifier: AuthenticAMD Family 23 Model 113 Stepping 0
	Microarchitecture: Zen 2
	Frequency (GHz): 3.79
	Number of physical packages: 1
	Number of physical CPUs: 6
	Number of logical CPUs: 12
	Graphics card #0 name: NVIDIA GeForce RTX 2070 SUPER
	Graphics card #0 vendor: NVIDIA (0x10de)
	Graphics card #0 VRAM (MB): 4095.00
	Graphics card #0 deviceId: 0x1e84
	Graphics card #0 versionInfo: DriverVersion=31.0.15.2824
	Memory slot #0 capacity (MB): 8192.00
	Memory slot #0 clockSpeed (GHz): 3.20
	Memory slot #0 type: DDR4
	Memory slot #1 capacity (MB): 8192.00
	Memory slot #1 clockSpeed (GHz): 3.20
	Memory slot #1 type: DDR4
	Memory slot #2 capacity (MB): 8192.00
	Memory slot #2 clockSpeed (GHz): 3.20
	Memory slot #2 type: DDR4
	Memory slot #3 capacity (MB): 8192.00
	Memory slot #3 clockSpeed (GHz): 3.20
	Memory slot #3 type: DDR4
	Virtual memory max (MB): 37540.23
	Virtual memory used (MB): 16555.55
	Swap memory total (MB): 4864.00
	Swap memory used (MB): 6.50
	JVM Flags: 4 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xss1M -Xmx16384m -Xms256m
	Loaded Shaderpack: (off)
	NEC status: No NEC detected
	Launched Version: forge-47.0.6
	Backend library: LWJGL version 3.3.1 build 7
	Backend API: NVIDIA GeForce RTX 2070 SUPER/PCIe/SSE2 GL version 4.6.0 NVIDIA 528.24, NVIDIA Corporation
	Window size: 1024x768
	GL Caps: Using framebuffer using OpenGL 3.2
	GL debug messages: 
	Using VBOs: Yes
	Is Modded: Definitely; Client brand changed to 'forge'; Server brand changed to 'forge'
	Type: Integrated Server (map_client.txt)
	Graphics mode: fancy
	Resource Packs: vanilla
	Current Language: fr_fr
	CPU: 12x AMD Ryzen 5 3600XT 6-Core Processor 
	Server Running: true
	Player Count: 1 / 8; [ServerPlayer['Un_Pain_Perdu'/56, l='ServerLevel[Aventure du Premier Pain]', x=-1013.05, y=117.00, z=185.99]]
	Data Packs: vanilla, mod:forge, mod:rubidium, mod:oculus, mod:clumps (incompatible), mod:journeymap (incompatible)
	Enabled Feature Flags: minecraft:vanilla
	World Generation: Stable
	ModLauncher: 10.0.9+10.0.9+main.dcd20f30
	ModLauncher launch target: forgeclient
	ModLauncher naming: srg
	ModLauncher services: 
		mixin-0.8.5.jar mixin PLUGINSERVICE 
		eventbus-6.0.3.jar eventbus PLUGINSERVICE 
		fmlloader-1.20.1-47.0.6.jar slf4jfixer PLUGINSERVICE 
		fmlloader-1.20.1-47.0.6.jar object_holder_definalize PLUGINSERVICE 
		fmlloader-1.20.1-47.0.6.jar runtime_enum_extender PLUGINSERVICE 
		fmlloader-1.20.1-47.0.6.jar capability_token_subclass PLUGINSERVICE 
		accesstransformers-8.0.4.jar accesstransformer PLUGINSERVICE 
		fmlloader-1.20.1-47.0.6.jar runtimedistcleaner PLUGINSERVICE 
		modlauncher-10.0.9.jar mixin TRANSFORMATIONSERVICE 
		modlauncher-10.0.9.jar fml TRANSFORMATIONSERVICE 
	FML Language Providers: 
		minecraft@1.0
		lowcodefml@null
		javafml@null
	Mod List: 
		client-1.20.1-20230612.114412-srg.jar             |Minecraft                     |minecraft                     |1.20.1              |DONE      |Manifest: a1:d4:5e:04:4f:d3:d6:e0:7b:37:97:cf:77:b0:de:ad:4a:47:ce:8c:96:49:5f:0a:cf:8c:ae:b2:6d:4b:8a:3f
		forge-1.20.1-47.0.6-universal.jar                 |Forge                         |forge                         |47.0.6              |DONE      |Manifest: 84:ce:76:e8:45:35:e4:0e:63:86:df:47:59:80:0f:67:6c:c1:5f:6e:5f:4d:b3:54:47:1a:9f:7f:ed:5e:f2:90
		rubidium-0.6.5.jar                                |Rubidium                      |rubidium                      |0.6.5               |DONE      |Manifest: NOSIGNATURE
		oculus-mc1.20-1.6.4.jar                           |Oculus                        |oculus                        |1.6.4               |DONE      |Manifest: NOSIGNATURE
		Clumps-forge-1.20.1-12.0.0.2.jar                  |Clumps                        |clumps                        |12.0.0.2            |DONE      |Manifest: NOSIGNATURE
		journeymap-1.20.1-5.9.9-forge.jar                 |Journeymap                    |journeymap                    |5.9.9               |DONE      |Manifest: NOSIGNATURE
	Crash Report UUID: 67f2dc01-2990-49a3-98f5-596c615083f6
	FML: 47.0
	Forge: net.minecraftforge:47.0.6
```

**Comment by mysticdrew** [comment]
This is likely an issue with JourneyMap, it is fixed in `JourneyMap-1.20.1-5.9.10-forge` should be on curseforge once approved(couple hours from now) and modrinth very soon (couple minutes)

**Comment by UnPainPerdu** [comment]
Ok, thanks
